Fastlane/Fastfile

```diff
@@ -0,0 +1,32 @@
+XC_PROJECT = File.absolute_path('../WebDriverAgent.xcodeproj')
+RETRIES = 3
+
+lane :test do
+  test_run_block = lambda do |testrun_info|
+    failed_test_count = testrun_info[:failed].size
+
+    if failed_test_count > 0
```

Comment:
You can use `if failed_test_count.positive?` to ensure greater than zero like `.zero?` (2.3+)

Reply:
`> 0` is also readable IMHO